README.md

```diff
@@ -23,7 +23,7 @@ ruggedized, waterproof, outdoor robotic rover applications.
 
 ### Brain
 
-See [brain/](/brain/) for instructions on building the `farm_ng`
+See [brain/](/docs/intelligence-kit/brain/brain-v2) for instructions on building the `farm_ng`
 lib for the brain and example applications for interacting with
 the [farm-ng brain](https://farm-ng.github.io/amiga-dev-kit/docs/brain/).
```

Comment:
I think both links here need to be fixed. Should be more like:

```
See [Brain ADK Install](/docs/brain/brain-install) for instructions on building the `farm_ng`
lib for the brain and example applications for interacting with
the [farm-ng brain](/docs/intelligence-kit/brain/brain-v2).
```

Also we should never have hard links to other pages on the site. Anytime you see something with either of these paths, please replace with a relative link
- https://amiga.farm-ng.com/
- farm-ng.github.io/amiga-dev-kit